drivers: memc: Add imx-flexspi-is66wvs8m8 driver

namjoshiniks opened the pull request:
Tested:
Verified that reading and writing data to the PSRAM with MCU's FlexSPI controller in Quad Mode, works as expected.

namjoshiniks committed:
drivers: memc: Add imx-flexspi-is66wvs8m8 driver

Tested:
Verified that reading and writing data to the PSRAM
with MCU's FlexSPI controller in Quad Mode, works as
expected.

Signed-off-by: Nikhil Namjoshi <nikhilnamjoshi@google.com>

DerekSnell reviewed (commented):
Hi @namjoshiniks ,
Thank you for contributing this driver.  Since this is a FlexSPI driver, it would be helpful to add the DTS settings you used as a reference to help enable others who want to use this driver.  I know there is no board in the upstream repo with this pSRAM device.  Perhaps you could add the DTS settings and FlexSPI clock frequency as a comment block in the driver source file.  Or at a minimum if there are issues adding to the source, include your DTS settings here in your PR.
Thanks

namjoshiniks committed:
drivers: memc: Fix copyright and add example DTS config

Example DTS config comment is meant to help users of the
driver, with the PSRAM device tree configuration.

Signed-off-by: Nikhil Namjoshi <nikhilnamjoshi@google.com>

namjoshiniks commented:
> Hi @namjoshiniks , Thank you for contributing this driver. Since this is a FlexSPI driver, it would be helpful to add the DTS settings you used as a reference to help enable others who want to use this driver. I know there is no board in the upstream repo with this pSRAM device. Perhaps you could add the DTS settings and FlexSPI clock frequency as a comment block in the driver source file. Or at a minimum if there are issues adding to the source, include your DTS settings here in your PR. Thanks

Done. Added the DTS config as a comment in the driver source.
